docs/getstarted/getting-started.md

```diff
@@ -0,0 +1,277 @@
+
+    > **Tip**: You can also select Color Themes from the VS Code Marketplace directly by selecting **Browse Additional Color Themes...** from the Color Theme picker.
+
+Beyond changing the color theme of the user interface, you can customize almost every part of VS Code by using settings. Maybe you prefer a different font for the editor, change word wrapping, or enable [floating editor windows](/docs/editor/custom-layout.md#floating-editor-windows).
+
+Let's use the **Settings Editor** to modify the tab size in the editor:
+
+1. Select **File** > **Preferences** > **Settings** (or press `kb(workbench.action.openSettings)`) to open the Settings Editor.
+
+    ![Screenshot that shows the Settings Editor.](images/getting-started/settings-editor.png)
+
+1. Modify the value of the **Editor: Tab Size** setting, and notice how the change is applied directly to your editor.
+
+    By default, when you change a setting, it applies to all your VS Code workspaces (**User Settings**). Switch to the **Workspace** tab to change a setting that only applies to the current workspace.
+
+    > **Tip**: Use the search box in the Settings Editor to quickly find a setting.
+
+There are many more ways to customize your VS Code experience, such as [changing default keyboard shortcuts](/docs/getstarted/keybindings.md), adding [code snippets](/docs/editor/userdefinedsnippets.md), or by adding extensions from the [Visual Studio Marketplace](#step-5-install-a-language-extension).
+
+## Step 4: Write some code
+
+VS Code is first and foremost a code editor, so let's start by writing some code! VS Code has built-in support for JavaScript, TypeScript, HTML, CSS, and more. In this tutorial, you create a sample JavaScript file and use some of the code editing features that VS Code offers.
+
+VS Code supports many programming languages and in the next step, you'll install a language extension to add support for a different language.
+
+1. In the Explorer view, create a new file `app.js`, and start typing the following JavaScript code:
+
+    ```javascript
+    function sayHello(name) {
+        console.log('Hello, ' + name);
+    }
+
+    sayHello('VS Code');
+    ```
+
+    As you type, you should see suggestions popping up that help you complete your code (*IntelliSense*). You can use the `kbstyle(Up)` and `kbstyle(Down)` keys to navigate the suggestions, and `kbstyle(Tab)` to insert the selected suggestion.
+
+    Notice also the formatting of the code (*syntax highlighting*), to help you distinguish between different parts of the code.
+
+    ![Screen capture that shows IntelliSense in action for a JavaScript file.](images/getting-started/javascript-intellisense.gif)
+
+1. Put the cursor on the `Hello,` string, select the lightbulb icon, and then select **Convert to template string** to use a Code Action to simplify the string concatenation.
```

Comment:
As a newer user, do I know what template string or simplified string concatenation mean?

Reply:
I'll add a note to clarify.